docs/dashboards/installation.md

```diff
@@ -0,0 +1,34 @@
+Installation
+===
+
+#### Install via NPM
+One of the ways of adding the Highcharts Dashboards to your web page is via npm script. You can get all Dashboards functionality in the package, which can be installed using following script:
+```bash
+npm install highcharts-dashboards
```

Comment:
Do we agree on that name? 
I'm not sure if this will be ready for MVP, perhaps we should leave it as a placeholder?

Reply:
ASFAIK we will add the dashboard to NPM when the official release will be available.

@pawelfus could you confirm?

Reply:
That's right, npm package will be published with the first release. Feel free to replace with a placeholder like `[PACKAGE-NAME]`